mapit_mysociety_org/templates/pricing.html

```diff
@@ -68,7 +68,7 @@ <h2 class="pricing__option__cost">£{{ PRICING.2.price }}/mth</h2>
                 {% else %}Max {{ PRICING.2.calls }} calls per month
                 {% endif %}
             </li>
-            <li><strong>Free</strong> use of our <a href="{% url "home" %}">CSV data mapper</a></li>
+            <li><strong>Free</strong> use of our <a href="{% url "home" %}">data mapper</a></li>
```

Comment:
I'm guessing the changes to copy in this file should be in another commit, or another PR?